pkg/token/token_test.go

```diff
@@ -0,0 +1 @@
+package token_test
```

Comment:
left over?

Reply:
Nah thanks for pointing this out. Forgot to add the tests. Add the tests both here and in the bootstrap provider.